src/traffic_crashlog/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@
 #
 #######################
 
-add_executable(traffic_crashlog procinfo.cc traffic_crashlog.cc)
+add_executable(traffic_crashlog procinfo.cc backtrace.cc traffic_crashlog.cc)
 
-target_link_libraries(traffic_crashlog PRIVATE ts::inkevent ts::records ts::tscore ts::tsapicore)
+target_link_libraries(traffic_crashlog PRIVATE ts::inkevent ts::records ts::tscore ts::tsapicore unwind::unwind)
```

Comment:
This wont work,  I had to manually add `unwind-ptrace unwind-generic`  on my ubuntu 20.04.